hledger/Hledger/Cli/Commands/Diff.txt

```diff
@@ -0,0 +1,19 @@
+diff
```

Comment:
./Shake commandhelp is the intended way to generate this file, maybe you did that already. If there's any differences I can fix.

Reply:
Ok, `./Shake commandhelp` fails for me:
```
pandoc -f markdown-smart-tex_math_dollars hledger/Hledger/Cli/Commands/Balancesheetequity.md --lua-filter tools/pandoc-dedent-code-blocks.lua -t plain > hledger/Hledger/Cli/Commands/Balancesheetequity.txt
Error running filter tools/pandoc-dedent-code-blocks.lua:
tools/pandoc-dedent-code-blocks.lua: openBinaryFile: does not exist (No such file or directory)
Error when running Shake build system:
  at src/Development/Shake/Internal/Args.hs:81:69-79:
* Depends on: commandhelp
  at Shake.hs:443:27-43:
* Depends on: hledger/Hledger/Cli/Commands/Balancesheetequity.txt
* Raised the exception:
user error (Development.Shake.cmd, system command failed
Command: /bin/sh -c 'pandoc -f markdown-smart-tex_math_dollars hledger/Hledger/Cli/Commands/Balancesheetequity.md --lua-filter tools/pandoc-dedent-code-blocks.lua -t plain > hledger/Hledger/Cli/Commands/Balancesheetequity.txt'
Exit code: 83
Stderr:
Error running filter tools/pandoc-dedent-code-blocks.lua:
tools/pandoc-dedent-code-blocks.lua: openBinaryFile: does not exist (No such file or directory)
)


```